app/decorators/podcast_episode_decorator.rb

```diff
@@ -34,4 +34,8 @@ def mobile_player_metadata
       podcastImageUrl: image_url
     }
   end
+
+  def published_at_int
```

Comment:
how's this move not breaking anything? 🔬 
I guess we call `.decorate` already somewhere :D

Reply:
Yeah, seems like we only call this in one place and it's after we use `PodcastEpisodeDecorator.decorate_collection`